cpu/stm32/stmclk/stmclk_gx.c

```diff
@@ -191,73 +220,83 @@ void stmclk_init_sysclk(void)
     /* disable all active clocks except HSI -> resets the clk configuration */
     RCC->CR = RCC_CR_HSION;
 
-#if CLOCK_LSE
-    stmclk_enable_lfclk();
+#if defined(CPU_FAM_STM32G0)
```

Comment:
```suggestion
#if defined(CLOCK_HSI_DIV)
```

Reply:
The HSI divider is something that only exists in G0, so the preprocessor check on G0 is more explicit here.